libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Generators/ActivityChecker.cs

```diff
@@ -0,0 +1,372 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AdaptiveCards;
+using Microsoft.Bot.Builder.LanguageGeneration;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Generators
+{
+    public static class ActivityChecker
+    {
+        /// <summary>
+        /// check the lg string result before generate an Activity.
+        /// </summary>
+        /// <param name="lgStringResult">string result from languageGenerator.</param>
+        /// <returns>Diagnostic list.</returns>
+        public static List<Diagnostic> Check(string lgStringResult)
```

Comment:
public method, public class. Please add an nulll/empty check on the string and throw an argument exception.